runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcMinimapOverlay.java

```diff
@@ -68,11 +68,11 @@ public Dimension render(Graphics2D graphics)
 	private void renderNpcOverlay(Graphics2D graphics, NPC actor, String name, Color color)
 	{
 		NPCComposition npcComposition = actor.getTransformedComposition();
-		if (npcComposition == null || !npcComposition.isInteractible())
+		if (npcComposition == null || !npcComposition.isInteractible()
+			|| (actor.isDead() && !config.highlightDeadNpcs()))
 		{
 			return;
 		}
```

Comment:
Re-add the line after this brace.
```suggestion
		}

```